status-server/src/index.js

```diff
@@ -25,11 +24,10 @@ app.use(bodyParser.urlencoded({
 }));
 
 app.use(express.static(publicDir));
-
-app.get('/'+config.prefix+':num\.:ext?', function (req, res) {
-  var num = req.params.num || 0;
-  var filename = path.join(dataDir, config.prefix + num);
-  var ext = req.params.ext || '';
+//handles JSON requests based on the :id status number. reads file of status number provided and returns the data in json format
+app.get('/api/user/:id/status', function(req, res){
```

Comment:
I want to keep the ability to have text/plain (user/:id/status), application/json (user/:id/status.json) and text/html (user/:id/status.html) responses. I'm fine with you breaking them out into their own distinct routes if you think that makes it clearer/cleaner.